node_modules/oae-core/termsandconditions/js/termsandconditions.js

```diff
@@ -21,6 +21,27 @@ define(['jquery', 'oae.core'], function($, oae) {
         var $rootel = $('#' + uid);
 
         /**
+         * Accept the invitation in the URL state, if any
+         */
+        var acceptInvitation = function() {
+            var invitationToken = oae.api.util.url().param('invitationToken');
```

Comment:
So what if you follow the invitation and then

a. Sign in with Twitter (which doesn't release an email address)
b. Sign in with Shibboleth/CAS that doesn't release a (valid) email address